TIDELIFT.rst

```diff
@@ -0,0 +1,60 @@
+========
+Tidelift
+========
+
+pluggy is a member of `Tidelift`_. This document describes how the core team manages
+Tidelift-related activities.
+
+What is it
+==========
+
+Tidelift aims to make Open Source sustainable by offering subscriptions to companies which rely
+on Open Source packages. This subscription allows it to pay maintainers of those Open Source
+packages to aid sustainability of the work.
+
+It is the perfect platform for companies that want to support Open Source packages and at the same
+time obtain assurances regarding maintenance, quality and security.
+
+Funds
+=====
+
+It was decided in the `mailing list`_ that the Tidelift contribution will be split evenly between
+members of the `contributors team`_ interested in receiving funding.
+
+The current list of contributors receiving funding are:
+
+* `@nicoddemus`_
+* `@The-Compiler`_
+* `@RonnyPfannschmidt`_
```

Comment:
@The-Compiler @RonnyPfannschmidt you OK with this?

Reply:
Sounds great!